File: vite-vue/src/openapi/models/CreateChoiceQuestionOptionDto.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ChoiceQuestionOptionIndex } from './ChoiceQuestionOptionIndex';

export type CreateChoiceQuestionOptionDto = {
    questionId?: string;
    content?: string | null;
    index?: ChoiceQuestionOptionIndex;
    isAnswer?: boolean;
};

File: asp.net core/src/Boc.ExamOnline.Application.Contracts/ChoiceQuestions/CreateChoiceQuestionOptionDto.cs
using Boc.ExamOnline.Exams.Enums;
using System;

namespace Boc.ExamOnline.ChoiceQuestions
{
    public class CreateChoiceQuestionOptionDto
    {
        /// <summary>
        /// 选项内容
        /// </summary>
        public string Content { get; set; }
        /// <summary>
        /// 选项序号
        /// </summary>
        public ChoiceQuestionOptionIndex Index { get; set; }
        public bool IsAnswer { get; set; }
    }
}
